src/app/games/siam/siam.component.html

```diff
@@ -29,40 +19,81 @@
-                 (click)="clickPiece(x, y)"
-                 [attr.transform]="'translate(' + ((x+1) * SPACE_SIZE) + ' ' + ((y+1) * SPACE_SIZE) + ')'"
-                 fill="brown" />
-            <g *ngIf="choosingOrientation(x, y)">
-                <use *ngFor="let orientation of ['RIGHT', 'DOWN', 'LEFT', 'UP']"
-                     id="chooseOrientation_{{ orientation }}"
-                     class="arrow selected-fill no-stroke"
-                     (click)="chooseOrientation(orientation)"
-                     xlink:href="#triangle"
-                     [attr.transform]="getArrowTransform(x+1, y+1, orientation)" />
-            </g>
+                 class="nonplayer"
+                 (click)="clickSquare(x, y)"
+                 [attr.transform]="'translate(' + ((x+1) * SPACE_SIZE) + ' ' + ((y+1) * SPACE_SIZE) + ')'" />
         </g>
     </g>
+    <g *ngFor="let coord of clickableCoords">
+        <rect *ngIf="coord.x === -1 || coord.x === 5 || coord.y === -1 || coord.y === 5"
+              id="square_{{ coord.x }}_{{ coord.y }}"
+              (click)="clickSquare(coord.x, coord.y)"
+              [attr.x]="SPACE_SIZE * (coord.x + 1)"
+              [attr.y]="SPACE_SIZE * (coord.y + 1)"
+              [attr.width]="SPACE_SIZE"
+              [attr.height]="SPACE_SIZE"
+              [ngClass]="getSquareClasses(coord.x, coord.y)"
+              class="base semi-transparent" />
+        <rect (click)="clickSquare(coord.x, coord.y)"
+              [attr.x]="SPACE_SIZE * (coord.x + 1)"
+              [attr.y]="SPACE_SIZE * (coord.y + 1)"
+              [attr.width]="SPACE_SIZE"
+              [attr.height]="SPACE_SIZE"
+              class="base no-fill small-stroke clickable" />
+    </g>
+    <rect *ngIf="selectedLanding.isPresent()"
+          (click)="clickSquare(selectedLanding.get().x, selectedLanding.get().y)"
+          [attr.x]="SPACE_SIZE * (selectedLanding.get().x + 1)"
+          [attr.y]="SPACE_SIZE * (selectedLanding.get().y + 1)"
+          [attr.width]="SPACE_SIZE"
+          [attr.height]="SPACE_SIZE"
+          class="base no-fill small-stroke selected" />
+    <rect *ngIf="selectedPiece.isPresent()"
+          (click)="clickSquare(selectedPiece.get().x, selectedPiece.get().y)"
+          [attr.x]="SPACE_SIZE * (selectedPiece.get().x + 1)"
+          [attr.y]="SPACE_SIZE * (selectedPiece.get().y + 1)"
+          [attr.width]="SPACE_SIZE"
+          [attr.height]="SPACE_SIZE"
+          class="base no-fill small-stroke selected" />
+    <g *ngFor="let player of Player.PLAYERS">
+        <g id="remainingPieces_{{ player.value }}"
+           (click)="selectPieceForInsertion(player)" >
+            <use *ngFor="let x of range(playerPieces(player))"
+                 class="base mid-stroke"
+                 [ngClass]="getRemainingPieceClasses(player, x)"
+                 xlink:href="#arrow"
+                 [attr.transform]="getRemainingPieceTransform(x, player)" />
+        </g>
+    </g>
```

Comment:
Je sais pas si ceux là sont monotag-able ?

Reply:
nope sinon on se retrouverait avec des id dupliqués, on veut l'id que sur le groupe

Reply:
Mauvais endroit pour le dire p-e mais, dans l'optique du "cliquer sur du sélectionner devrais dé-selectionner" -- sur laquelle je suis dans un autre ticket (ou je ne touche pas Siam naturellement) -- on devrais pas appliquer ça sur les remaining pieces du Siam ? C'pas une énorme lacune UX mais c'est sexy.

Après je crois que tu peux faire <g *ngFor="let player of players" id="joueur_{{ player.value }}" (click)="cliquezLe(player)"> Et passer à seulement deux valises !

Reply:
ok pour le reclic sur les pièces (TODO)

Le "après je crois que tu peux faire ...", c'est pas ce qui est fait exactement

Reply:
Non y a un g non-nécessaire au milieu :p

Reply:
ah oui mes yeux avaient déjà mergé :D Corrigé !